code/world.dm

```diff
@@ -657,6 +661,11 @@ var/f_color_selector_handler/F_Color_Selector
 	SPAWN_DBG(10 SECONDS)
 		Reboot_server()
 #endif
+#ifdef UNIT_TESTS
+	SPAWN_DBG(10 SECONDS) // 10 SECONDS ?
```

Comment:
`?`

Reply:
This is the arbitrary amount of time to let the server run until telling it fuck of because there was probably a runtime.

Reply:
but why is the same exact time commented right next to the spawn

Reply:
Hmmm, poorly identified todo of if 10 seconds made sense.